tokio/src/sync/broadcast.rs

```diff
@@ -2,8 +2,8 @@
 //! all consumers.
 //!
 //! A [`Sender`] is used to broadcast values to **all** connected [`Receiver`]
-//! values. Both [`Sender`] and [`Receiver`] handles are clone-able, allowing
-//! concurrent send and receive actions. Additionally, [`Sender`] is `Sync`.
+//! values. [`Sender`] handles are clone-able, allowing concurrent send and
+//! receive actions. [`Sender`] and [`Receiver`] are both `Send` and `Sync`.
```

Comment:
`Receiver` is only Send and Sync if `T` is. I would also mention that you can use `Sender::subscribe` to get a new `Receiver`.

Reply:
Updated to include language around `T` in cb502b1. The 4th paragraph mentions using `Sender::subscribe`. to get new `Receiver` instances.